src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
@jonsterling The warnings are very useful because I actually fixed bugs by reviewing all the unused variables. For example, this time, one of the goals in S1-elim was created but forgotten in the returned expression, which was detected by the warnings. I also feel the ugliness of the code, but since bugs were found I feel the price is small.